app/assets/stylesheets/application_layout.scss

```diff
@@ -103,7 +103,10 @@ body, p, ol, ul, td, a {
 }
 
 #application-footer {
-  clear: both;
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
 
```

Comment:
Note: The mockups didn't show any copyright info or footer.  I left the existing one in place bug locked it to the bottom of the page.